README.md

```diff
@@ -10,27 +10,32 @@ A Remote-Settings Client built using Rust to read collection data.
 
 ## Example
 
-Below example uses [Tokio](https://tokio.rs) and utilizes some optional features, so your `Cargo.toml` could look like this:
+Below example uses [Viaduct](https://github.com/mozilla/application-services/tree/master/components/viaduct) and [Viaduct-Reqwest](https://github.com/mozilla/application-services/tree/master/components/support/viaduct-reqwest), so your `Cargo.toml` could look like this:
 
 ```toml
 [dependencies]
 tokio = { version = "0.2.21", features = ["macros", "tcp"] }
 log = "0.4.0"
 env_logger = "0.7.1"
+viaduct = { git = "https://github.com/mozilla/application-services", rev = "61dcc364ac0d6d0816ab88a494bbf20d824b009b"}
+viaduct-reqwest = { git = "https://github.com/mozilla/application-services", rev = "61dcc364ac0d6d0816ab88a494bbf20d824b009b"}
```

Comment:
Either now or later, this needs to be an optional dependency, so that consumers can plug in their preferred viaduct implementation.  For now, maybe just make this a dev dependency?

Reply:
done